ow_power_system/include/PowerSystemNode.h

```diff
@@ -182,10 +191,12 @@ class PowerSystemNode
   bool m_icl_activated = false;
   bool m_thermal_failure_activated = false;
 
+  double m_mean_mechanical_power = 0.0;
+  double m_electrical_power = 0.0;
+
   // Used for mechanical power debug printouts.
   double m_power_watts;
-  double m_mean_mechanical_power;
-  double m_sum_mechanical_power = 0;
+  double m_sum_mechanical_power = 0.0;
```

Comment:
Although this existed before, it should not be a member variable, but rather a local variable in jointStatesCb().

Reply:
This needs to be a member variable. In the `jointStatesCb()` function it's being operated on by a `+=`.

Reply:
That was my point.  I see it used only in that function, so it can be local to that function.

Reply:
It's irrelevant whether its only used in a single function, it carries state between function calls. It accumulates a value each time. Making it local would break things.